### PR TITLE
Option to show all articles in the new mod_articles

### PR DESCRIPTION
**Pull request by RickR2H**
Pull Request for Issue # .

### Summary of Changes
With #43738 a new articles module is added. This new module mod_articles module has no option to show all articles. In the mod_articles_category for instance the "Articles to Display" can be set to 0 to show all articles. This PR add the option to set the "Articles to Display" to 0 to show all articles.

### Testing Instructions
- Set the "Articles to Display" to 0 to show all articles matching the settings.
- New all articles are shown.

### Actual result BEFORE applying this Pull Request.
The option "Articles to Display" cannot be set to 0 to show all articles.

### Expected result AFTER applying this Pull Request
The option "Articles to Display" can be set to 0 to show all articles.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by drmenzelit** [comment]
I have tested this item :white_check_mark: successfully on 35c4a025e4428ac982eff8ac8dadc5e0db559893<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44127">issues.joomla.org/tracker/joomla-cms/44127</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 35c4a025e4428ac982eff8ac8dadc5e0db559893<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44127">issues.joomla.org/tracker/joomla-cms/44127</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44127">issues.joomla.org/tracker/joomla-cms/44127</a>.</sub>

File: language/en-GB/mod_articles.ini [hunk 1]
```diff
@@ -15,7 +15,7 @@ MOD_ARTICLES_FIELD_AUTHOR_LABEL="Authors"
 MOD_ARTICLES_FIELD_CATDEPTH_LABEL="Category Depth"
 MOD_ARTICLES_FIELD_CATFILTERINGTYPE_DESC="Include or exclude the selected categories."
 MOD_ARTICLES_FIELD_CATFILTERINGTYPE_LABEL="Category Filtering Type"
-MOD_ARTICLES_FIELD_COUNT_DESC="The number of items to display. The default value of 0 will display all articles."
+MOD_ARTICLES_FIELD_COUNT_DESC="The number of items to display. Set the value to 0 to show all articles."
```
Comment (Quy) [inline language/en-GB/mod_articles.ini]:
```suggestion
MOD_ARTICLES_FIELD_COUNT_DESC="The number of items to display. Set to 0 to show all articles."
```

**Comment by Hackwar** [comment]
Thanks for your contribution @RickR2H!

**Pull request merged**